src/test/java/core/basesyntax/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.exception.WritingException;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FileWriterServiceImplTest {
+    private FileWriterServiceImpl fileWriterService = new FileWriterServiceImpl();
+
+    @Test
+    void write_nullData_NotOk() {
+        String fileName = "testFile";
+        assertThrows(WritingException.class, () -> fileWriterService.write(fileName, null));
+    }
+
+    @Test
+    void write_okData_Ok() throws Exception {
+        String fileName = "src/test/resources/testFile";
+        String expectedString = "onetwo";
+        Path path = Paths.get(fileName);
+        List<String> stringList = Files.readAllLines(path);
+        StringBuilder stringBuilder = new StringBuilder();
+        stringList.forEach(stringBuilder::append);
+        String actualString = stringBuilder.toString();
```

Comment:
let's not do these extra steps in order to validate the method output. It's better to validate the result list with the expected list. In case you want to create a single string from a list it's better to use String.join()